src/languageclient/startArmLanguageServer.ts

```diff
@@ -27,15 +27,13 @@ export enum LanguageServerState {
     Started,
     Stopped,
 }
-let client: LanguageClient | undefined;
-
 export async function stopArmLanguageServer(): Promise<void> {
     ext.outputChannel.appendLine(`Stopping ${languageServerName}...`);
     // Work-around for https://github.com/microsoft/vscode/issues/83254 - store languageServerState global via ext to keep it a singleton
     ext.languageServerState = LanguageServerState.Stopped;
-    if (client) {
-        await client.stop();
-        client = undefined;
+    if (ext.languageServerClient) {
+        await ext.languageServerClient.stop();
+        ext.languageServerClient = undefined;
```

Comment:
These two lines should probably be reversed.

Reply:
(using a temp var) - and I realize, not your code :-)

Reply:
Sure, that makes sense, since we now have external users.